src/TableFunctions/TableFunctionS3Cluster.h

```diff
@@ -13,7 +13,7 @@ namespace DB
 class Context;
 
 /**
- * s3Cluster(cluster_name, source, [access_key_id, secret_access_key,] format, structure)
+ * S3 cluster(cluster_name, source, [access_key_id, secret_access_key,] format, structure)
```

Comment:
Previous version was correct, this one incorrect

Reply:
OK, thanks.